lib/bloc/contact_bloc.dart

```diff
@@ -0,0 +1,20 @@
+import 'contact_interface_bloc.dart';
+import '../model/contact.dart';
+import '../repository/repository_interface_contact.dart';
+
+class ContactBloc extends ContactInterfaceBloc {
```

Comment:
As an extra good practice, you should put the interfaces into a new folder, called "interface", and apply it to the repositories and the services as well

Reply:
done